src/lattice/lattice.js

```diff
@@ -10,6 +10,8 @@ import {LatticeVectors} from "./lattice_vectors";
 import {primitiveCell} from "../cell/primitive_cell";
 import {Cell} from "../cell/cell";
 
+export const nonPeriodicLatticeAdditiveValue = 2.0;
```

Comment:
(1) we could also call this `nonPeriodicLatticeMinimumSizeAngstroms` 
(2) let's add a comment explaining what this is
(3) I think it should be used in https://github.com/Exabyte-io/exabyte-web-app/pull/1660/files# also